src/lib/components/quiz/more-info.svelte

```diff
@@ -0,0 +1,6 @@
+<script lang="ts">
+  export let teamPage = 'help';
```

Comment:
This component will show more info given a team match.
```suggestion
  import type { TeamMatch } from '$lib/constants/quiz';

  export let teamMatch: TeamMatch = TeamMatch.TEAMLESS;
```

Reply:
Done, unnecessary after final implementation